Tools/parametric_model/generate_parametric_model.py

```diff
@@ -78,13 +76,13 @@ def start_model_estimation(config, log_path, data_selection="none", plot=False,
                     "directory and models/__init__.py?".format(model_class)
         raise AttributeError(error_str)
 
-    model.load_dataframes(data_df)
-    model.prepare_regression_matrices()
-    model.compute_fisher_information()
-
```

Comment:
It might seem a bit strange, but we actually want to do this before we do the data selection.

Computing the fisher information before you run the data selection helps you to determine which part of the log is more informative than the others :)